packages/sdk-themes/src/themes/monite.ts

```diff
@@ -318,6 +326,26 @@ export const defaultMoniteComponents: Components<Omit<Theme, 'components'>> = {
             },
           },
         },
+        '&.Monite-SearchField': {
+          maxWidth: '400px',
+          width: '100%',
+        },
+      },
+    },
+  },
+  MuiStack: {
+    styleOverrides: {
+      root: {
+        '&.Monite-Filters': {
+          '& > *': {
```

Comment:
@radist2s I made this changes in theme itself, but the filters will look worse in the blank Material UI theme. I can move the layout styles to the sdk itself.